app/src/main/java/io/homeassistant/android/api/results/Entity.java

```diff
@@ -40,19 +43,28 @@
     public String last_changed;
     public String last_updated;
     public String state = null;
-    public Attributes attributes;
+
+    public Ason attributes;
 
     @Override
     public String toString() {
         return String.format("%1$s[%2$s]", type, id);
     }
 
+    public boolean isHidden() {
+        return (attributes.has("hidden") && (((Boolean)attributes.get("hidden")).booleanValue() == true));
```

Comment:
This method seems to be overly complicated. If a view doesn't have the parameter, it isn't hidden, so `return attributes.getBool("hidden");` is enough.